mobilitydb/sql/point/052_tpoint.in.sql

```diff
@@ -310,13 +311,13 @@ CREATE FUNCTION tempSubtype(tgeogpoint)
   AS 'MODULE_PATHNAME', 'Temporal_subtype'
   LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
 
-CREATE FUNCTION interpolation(tgeompoint)
+CREATE FUNCTION interp(tgeompoint)
   RETURNS text
-  AS 'MODULE_PATHNAME', 'Temporal_interpolation'
+  AS 'MODULE_PATHNAME', 'Temporal_interp'
   LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
-CREATE FUNCTION interpolation(tgeogpoint)
+CREATE FUNCTION interp(tgeogpoint)
   RETURNS text
-  AS 'MODULE_PATHNAME', 'Temporal_interpolation'
+  AS 'MODULE_PATHNAME', 'Temporal_interp'
```

Comment:
I can understand shortening 'interpolation' to 'interp' in the code, but would it not be better to keep the SQL function as `interpolation` for clarity?